pkgs/desktops/gnome-3/extensions/gsconnect/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, libxml2, pkgconfig
+, glib, folks, nautilus, libcanberra, sshfs }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-gsconnect-${version}";
+  version = "13-alpha";
+
+  src = fetchFromGitHub {
+    owner = "andyholmes";
+    repo = "gnome-shell-extension-gsconnect";
+    rev = "v${version}";
+    sha256 = "0dynq5b2vrp54r9k6fpc81qjmr2qv1nbbyn4khrjxv283q7w3h7q";
+  };
+
+  # TODO: Missing "dbus-1"
```

Comment:
D-Bus is not required – it is only used for obtaining install path, but there is a [more convenient fallback](https://github.com/andyholmes/gnome-shell-extension-gsconnect/blob/dccdfa1a63e33f505e38f7d2667c79ff6addd8b1/meson.build#L90).